app/main.py

```diff
@@ -43,17 +44,16 @@
 def startup_event():
     db.wait_for_redis()
 
-    db.initialize()
     picks.initialize()
     verification.initialize()
 
 
 @app.post("/update")
 def update():
-    new_apps = apps.load_appstream()
+    app_downloads = stats.update()
```

Comment:
Not sure if I follow what happens here?

Reply:
Running stats updates first, so that we can generate a list of appId + downloads last 30 days

Then we can just use that list when we do `load_appstream`. Stats has `EOL` apps and so does our download list. But as we run `load_appstream`, we can just access our key/values via an `appId` from the appstream. So this makes sure we don't create search entries for apps that are not in the appstream.

Reply:
Had to revert this, as stats already depend on data from the appstream, as it accesses the database to figure out how many apps there are.